### PR TITLE
chore: setting firebase environments

### LINKED ISSUE #39: setting firebase environements

### PR DESCRIPTION
**Pull request by kobakaku**
close #39

### CODE REVIEW

**Comment by kobakaku** [comment]
Please try to merge this branch !!
I will check if Github Actions work correctly .

**Comment by kobakaku** [comment]
You can see that e2e test works !

Please try merging this branch to main and check cd-firebase-testnet.yml !!

**Comment by YasunoriMATSUOKA** [comment]
@kobakaku 

I guess CI Web Angular E2E fails on https://github.com/next-web-technology/symbol-social-wallet/tree/feature/setting-firebase-environments

ex. https://github.com/next-web-technology/symbol-social-wallet/actions/runs/4485195103

Can you fix that?

It seems to be not counted as checks because it is deployment or invalid yml file.

File: projects/web/src/environments/environment.example.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  production: false,
+  useEmulators: true,
+  // Put the information of firebaseConfig to GitHub
```
Comment (YasunoriMATSUOKA) [inline projects/web/src/environments/environment.example.ts:4]: We need to add only key into example env file.
But do not add real value of firebase config values.

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
- Fix GitHub Actions yml files and check them to pass.
- Add only keys related firebase config. (Do not add real values related firebase config. Value should be dummy. (ex. `""`))

File: .github/workflows/ci-web-angular-build.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ jobs:
   web-angular-build:
     runs-on: ubuntu-latest
 
+    environment:
+      name: testnet
```
Comment (YasunoriMATSUOKA) [inline .github/workflows/ci-web-angular-build.yml:12]: With this, testnet environment secret is used.
I guess you need to remove this to change this github actions from deployment to check.

File: .github/workflows/ci-web-angular-e2e.yml [hunk 3]
```diff
@@ -8,6 +8,9 @@ jobs:
   web-angular-e2e:
     runs-on: ubuntu-latest
 
+    environment:
+      name: testnet
```
Comment (YasunoriMATSUOKA) [inline .github/workflows/ci-web-angular-e2e.yml:12]: With this, testnet environment secret is used.
I guess you need to remove this to change this github actions from deployment to check.

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
Check review comments.

**Review by YasunoriMATSUOKA — APPROVED** [review]
LGTM

**Pull request merged**